tests/integrations/document/test_document.py

```diff
@@ -148,3 +149,25 @@ def test_create_doc_from_dict():
     doc2 = MyDoc(text='txt', other='also text')
 
     assert isinstance(doc1, BaseDoc) and isinstance(doc2, BaseDoc)
+
+
+def test_create_from_named_tuple():
```

Comment:
in your test can u add a line where you instantiated a Doc ?

Reply:
@h4shk4t please add this as well

Reply:
Done!